src/apps/LandingRegister/Footer.react.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
```

Comment:
this component could be stateless
https://facebook.github.io/react/docs/reusable-components.html#stateless-functions